Source/Scene/Expression.js

```diff
@@ -273,6 +273,12 @@ define([
             node = new Node(ExpressionNodeType.CONDITIONAL, '?', left, right, test);
         } else if (ast.type === 'MemberExpression') {
             node = parseMemberExpression(expression, ast);
+        } else if (ast.type === 'ArrayExpression') {
+            var val = [];
+            for (var i=0; i < ast.elements.length; i++) {
```

Comment:
Minor whitespace.  Throughout this file and in general, use something like: `for (var i = 0;`